src/handy-work.js

```diff
@@ -47,22 +56,26 @@ class HandInfo {
 }
 const hands = new Map();
 
-let __dumpHands = false;
+function resetHands() {
+	hands.clear();
+}
+
+window.__dumpHands = false;
 function dumpHands() {
-	__dumpHands = true;
+	window.__dumpHands = true;	
 }
 
+let tempHands = {};
 function handDataToFile(inputSources, referenceSpace, frame) {
 
-	const hands = {};
 	for (const source of inputSources) {
 		if (!source.hand) continue;
-		hands[source.handedness] = source.hand;
+		tempHands[source.handedness] = source.hand;
```

Comment:
Why do you have a global tempHands object instead of a local closure hands?

It doesn't seem to be used else where and it's not a performance improvement.